tests/Elasticsearch/Tests/YamlRunnerTest.php

```diff
@@ -328,6 +328,9 @@ public function operationDo($operation, $lastOperationResult, &$context, $testNa
             $endpointParams->client['headers'] = $headers;
         }
 
+        if (!is_string($method)) {
```

Comment:
It may not be a string at this point and `$this->mapEndpoint` would fail, hence the added check.